client/my-sites/people/invite-people/index.jsx

```diff
@@ -89,8 +93,18 @@ const InvitePeople = React.createClass( {
 			return value;
 		} );
 
+		const filteredErrors = pickBy( errors, ( error, key ) => {
```

Comment:
Because, with errors, we get the value as well as an explanation as to why the validation failed. With successes, we don't need the explanation.